LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceFactory.cs

```diff
@@ -135,8 +135,11 @@ public virtual ILoRaDeviceClient CreateDeviceClient(string deviceId, string prim
                         AmqpConnectionPoolSettings = new AmqpConnectionPoolSettings()
                         {
                             Pooling = true,
-                            // pool size 1 => 995 devices.
-                            MaxPoolSize = 1
+                            // pool size for this project defaults to 1, allowing aroung 1000 amqp links
+                            // in case you need more, and the communications are proxied through edgeHub,
+                            // please consider changing this parameter in edgeHub configuration too
+                            // https://github.com/Azure/iotedge/blob/e6d52d6f6b0eb76e7ef250f3fcdeaf38e467ab4f/doc/EnvironmentVariables.md
+                            MaxPoolSize = this.configuration.IotHubConnectionPoolSize
```

Comment:
Do we need to add this change to the docs as well?

Reply:
We currently don't have a documentation page explaining all the possible environment variables accepted for LNS module.
I will create an item for improving the documentation in the future